bootstrap.js

```diff
@@ -18,14 +19,19 @@ module.exports = function(grunt) {
   }, buildPaths);
   grunt.config('config.buildPaths', buildPaths);
 
-  // Wrap Grunt's loadNpmTasks() function to change the current directory to
-  // grunt-drupal-tasks, so that module dependencies of it are found.
+  // Wrap Grunt's loadNpmTasks() function to allow loading Grunt task modules
+  // that are dependencies of Grunt Drupal Tasks.
   grunt._loadNpmTasks = grunt.loadNpmTasks;
   grunt.loadNpmTasks = function (mod) {
-    var pathOrig = process.cwd();
-    process.chdir(__dirname);
+    var internalMod = grunt.file.exists(path.join(__dirname, 'node_modules', mod));
```

Comment:
It looks like `grunt.file.exists` will call `path.join` on all its arguments, so this one may be superfluous: https://github.com/gruntjs/grunt/blob/c6b534a7bc2e0a521f1fccf4963e58d58d3009e3/lib/grunt/file.js#L399